crates/matrix-crypto-ffi/src/machine.rs

```diff
@@ -517,7 +535,7 @@ impl OlmMachine {
             content: &'a RawValue,
         }
 
-        let event: SyncMessageEvent<RoomEncryptedEventContent> = serde_json::from_str(event)?;
+        let event: SyncMessageLikeEvent<RoomEncryptedEventContent> = serde_json::from_str(event)?;
```

Comment:
I think this would be a bit easier to read when using the type alias from the event module, i.e. `ruma::events::room::encrypted::SyncRoomEncryptedEvent`.

Reply:
Did in https://github.com/matrix-org/matrix-rust-sdk/pull/557/commits/64273fe63930d1ca8c923da0d0e4cb1b9fdc60ff.